wdae/wdae/studies/remote_study.py

```diff
@@ -93,7 +93,7 @@ def description(self):
 
     @property
     def is_group(self):
```

Comment:
Please add tests for this method.

Reply:
Adding the tests proved troublesome because of remote dataset support, so now remote genotype datas cannot be groups.